plugins/dicom_viewer/webpack.helper.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(x) {
+    x.module.loaders.push({
+        test: /\.json$/,
+        loader: 'json',
+        include: ['node_modules'],
```

Comment:
I had trouble getting that to work even the second time around. I can try again.